### ISSUE TITLE
Individual files

### ISSUE THREAD

**Issue by ghost** [k0shk0sh/FastHub#2910]
**FastHub Version: 4.7.3**  
**Android Version: 9 (SDK: 28)**  
**Device Information:**  
- **Manufacturer:** samsung  
- **Brand:** samsung  
- **Model:** SM-A107F  
---

 I want to download individual files from respotary

**Comment by l3laze** [comment]
Non-binary files, i.e text or code, can be downloaded: Go to the repo and tap the file to view it, tap the 3-dot menu, and then press "Download". No idea if this works for binaries, but it very well might.


On some versions of Android this may get dropped in the Downloads folder, and others I believe it would be saved in `Android/data/com.fastaccess.fasthub/files`.
